minecraft_servers/crystopia/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+  "server_name": "crystopia",
+  "nice_name": "Crystopia",
+  "direct_ip": "play.crystopia.net",
+  "server_wildcards": [
+    "mc.crystopia.net"
+  ],
+  "social": {
+    "web": "https://crystopia.net",
+    "web_shop": "none",
+    "web_support": "https://crystopia.net/support",
+    "twitter": "CrystopiaNet",
+    "youtube": "https://www.youtube.com/@",
```

Comment:
You forgot the tag ig. Or just remove it. If I would merge your PR, our users would get maany wrong redirects.. 😬😬